CentOS.md

```diff
@@ -61,19 +120,13 @@ For a debug/test build:
 ```
 For a release build:
 ```
-../Qt/5.12.4/gcc_64/bin/qmake ../qgroundcontrol/qgroundcontrol.pro spec linux-g++ CONFIG+=qtquickcompiler
+../Qt/5.12.4/gcc_64/bin/qmake ../qgroundcontrol/qgroundcontrol.pro -spec linux-g++ CONFIG+=qtquickcompiler
```

Comment:
@dogmaphobic I was assuming there needs to be a dash here since the debug command contains it and I was successfully using the command iwth the dash for both builds.

Reply:
Yes, the missing dash is a typo.